src/Compilers/VisualBasic/Portable/Symbols/Retargeting/RetargetingNamedTypeSymbol.vb

```diff
@@ -546,5 +546,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Retargeting
                 Yield RetargetingTranslator.Retarget(underlying)
             Next
         End Function
+
+        Friend Overrides ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
+            Get
+                Dim underlyingRequiredMembers = _underlyingType.AllRequiredMembers
+                Dim builder = ImmutableSegmentedDictionary.CreateBuilder(Of String, Symbol)()
+                For Each kvp In underlyingRequiredMembers
+                    builder.Add(kvp.Key, RetargetingTranslator.Retarget(kvp.Value))
+                Next
+
+                Return builder.ToImmutable()
```

Comment:
>Return builder.ToImmutable() [](http://example.com/codeflow?start=16&length=28)

I think all retargeting symbols are wrappers around symbols from VB module. So, they cannot have any required member "things". Instead of having "dead" code, let's assert that underlying symbols return false and empty sets form the new APIs and explicitly return false and empty sets.